web/apps/labelstudio/src/pages/Settings/GeneralSettings.jsx

```diff
@@ -28,7 +28,8 @@ export const GeneralSettings = () => {
   return (
     <Block name="general-settings">
       <Elem name={"wrapper"}>
-        <h1>General Settings</h1>
+        <h1>General Settings!</h1>
+
```

Comment:
```suggestion
        <h1>General Settings</h1>
```

Looks like example changes that were committed on accident.